bedrock/firefox/views.py

```diff
@@ -22,6 +22,7 @@
 from lib import l10n_utils
 from lib.l10n_utils.dotlang import _
 from product_details.version_compare import Version
+import waffle
```

Comment:
Would it be a worthwhile optimization to do `from waffle import switch_is_active` here? Not sure how much difference it makes, but perhaps @pmclanahan or @jgmize might have a preference.